.gitlab-ci.yml

```diff
@@ -39,6 +39,16 @@ synopsys_dc:
         $CI_PROJECT_DIR/.gitlab-ci.d/memora_retry.sh insert synopsys_dc
       fi
 
+fuse_xsim:
+  stage: build
+  script:
+    - bender sources
+    - /usr/local/anaconda3/bin/python3 -m pip install fusesoc --user
+    - mkdir fusesoc && cd fusesoc
+    - $HOME/.local/bin/fusesoc library add axi ..
+    - $HOME/.local/bin/fusesoc core list
+    - vitis $HOME/.local/bin/fusesoc run --tool xsim --target sim --no-export pulp-platform.org::axi:$(cat ../VERSION | sed s/-/./g)
```

Comment:
we should add a version here and use the newest one installed (`vitis-2022.1-zr`)